moabb/datasets/bnci.py

```diff
@@ -642,49 +642,52 @@ def data_path(self, subject, path=None, force_update=False,
 class BNCI2014001(MNEBNCI):
     """BNCI 2014-001 Motor Imagery dataset"""
 
-    def __init__(self, tmin=3.5, tmax=5.5):
+    def __init__(self, tmin=0.5, tmax=4):
```

Comment:
according to this : https://lampx.tugraz.at/~bci/database/001-2014/description.pdf
the motor imagery start at 3 second after the beginning of the trial (trigger).
I dont get you new your new parameters

Reply:
i get a decrease in perf with this parameters

Reply:
yep, the beep is at the trigger , the cue at 2 s, and the end around 6 s (epoch average at based on MNE epoching from the events)

![image](https://user-images.githubusercontent.com/2145718/38404829-b19689e4-393a-11e8-8cbc-0d7e4a07f356.png)

Reply:
Sorry--you're probably right, I was confused because I didn't know if the trigger was at the cue or the beep (I think in one other dataset it's the cue...its the one where I commented that the markers are task start not trial start)